methods.go

```diff
@@ -42,6 +42,20 @@ func (c *Client) GetFreeSpace(path string) (int64, error) {
 	return freeSpace, nil
 }
 
+// GetListenPort return the listen port of daemon session
+func (c *Client) GetListenPort() (port uint16, err error) {
```

Comment:
Please do not use named return values, they are not used anywhere else.

Reply:
Oh, I see.